src/hosting.test.js

```diff
@@ -1,12 +1,16 @@
 "use strict";
 
 import fs from "fs";
+import https from "https";
 import path from "path";
 
 import pagexray from "pagexray";
```

Comment:
I can't remember, but I think we might no longer need `pagexray` here, as @soulgalore mentioned making a release to move functionality into his library. This would mean we could remove pagexray as discussed it in #182. 

https://github.com/thegreenwebfoundation/co2.js/issues/182

Knowing Peter, I reckon he'd be one the first to add a user agent to requests too, once this feature is merged in 😄 

@fershad is it ok to leave this call with you? It's small change, but it might be betteer to handle outside of this PR.

Reply:
Cheers Chris, we'll remove the `pagexray` stuff in the next minor release of CO2.js so not this update.  I'll create a separate PR to track that bit of work.

> Knowing Peter, I reckon he'd be one the first to add a user agent to requests too, once this feature is merged in 😄

Hahah I thought the same thing ~~